components/case-studies/CaseStudiesCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Image from 'next/image';
+
+import { CaseStudiesCardProps } from '@/types';
+
+const CaseStudiesCard: React.FC<CaseStudiesCardProps> = ({
+  altText,
+  projectName,
+  heading,
+  subHeading,
+  image,
+  _id,
+}) => {
+  const projectBackgrounds: Record<string, string> = {
+    morrent: 'bg-morrent-background',
+    jobit: 'bg-jobit-background',
+    filmpire: 'bg-filmpire-background',
+  };
```

Comment:
Please move this into a separate file. All of your hard-coded data that is used should be in one root file and then imported when/where necessary. 

- Once you import your data, a better approach to consider is to map over your data instead of hard-coding it. This way, you can return a `<CaseStudyCard>` instead. It's a much cleaner approach that improves readability and maintainability.